backpack/custom_module/graph_utils.py

```diff
@@ -101,16 +114,24 @@ def _transform_mul_to_scale_module(module: Module, debug: bool) -> GraphModule:
     Raises:
         RuntimeError: if a multiplication is found but node.args are not (float, Node)
     """
-    target = "<built-in function mul>"
+    target_function = "<built-in function mul>"
+    target_method = "multiply"
     if debug:
-        print(f"\tBegin transformation: {target} -> ScaleModule")
+        print(f"\tBegin transformation: {target_function} -> ScaleModule")
```

Comment:
also add `target_method`

Reply:
Who is the target of debug messages?
- Users? Then I don't think they care about the difference between function/method
- Developers? Yes, then I'll add all the targets.